src/app/common/modal/container-modal/container-modal.component.scss

```diff
@@ -0,0 +1,12 @@
+@import '../../../../styles/shared';
+
+$box-shadow-color: rgba($ux-color-modal-header, 0.5);
+
+:host,
+div[cdkTrapFocus] {
+	display: contents;
+}
```

Comment:
Bootstrap already supports the scrolling behavior we wanted, but extra elements were preventing the styles from being applied properly.  This fixes that.